src/MSRIOGroup.cpp

```diff
@@ -226,6 +201,79 @@ namespace geopm
+                max_turbo_name = "MSR::TURBO_RATIO_LIMIT:MAX_RATIO_LIMIT_1CORE";
+                break;
+            case MSRIOGroup::M_CPUID_SKX:
+            case MSRIOGroup::M_CPUID_ICX:
+                max_turbo_name = "MSR::TURBO_RATIO_LIMIT:MAX_RATIO_LIMIT_0";
+                break;
+            default:
+                throw Exception("MSRIOGroup: Unsupported CPUID",
+                                GEOPM_ERROR_RUNTIME, __FILE__, __LINE__);
+        }
+
+        if (m_hwp_is_enabled)
+        {
+            register_signal_alias("FREQUENCY_MAX", "MSR::HWP_CAPABILITIES:HIGHEST_PERFORMANCE"); // TODO: Remove @ v2.0
+            register_signal_alias("CPU_FREQUENCY_MAX", "MSR::HWP_CAPABILITIES:HIGHEST_PERFORMANCE");
+
+            register_signal_alias("FREQUENCY_MIN", "MSR::HWP_CAPABILITIES:LOWEST_PERFORMANCE"); // TODO: Remove @ v2.0
+            register_signal_alias("CPU_FREQUENCY_MIN", "MSR::HWP_CAPABILITIES:LOWEST_PERFORMANCE");
+
+            register_signal_alias("FREQUENCY_GUARANTEED", "MSR::HWP_CAPABILITIES:GUARANTEED_PERFORMANCE"); // TODO: Remove @ v2.0
+            register_signal_alias("CPU_FREQUENCY_EFFICIENT", "MSR::HWP_CAPABILITIES:MOST_EFFICIENT_PERFORMANCE");
+        }
+        else {
+            register_signal_alias("FREQUENCY_MAX", max_turbo_name); // TODO: Remove @ v2.0
+            register_signal_alias("CPU_FREQUENCY_MAX", max_turbo_name);
+        }
+
+        set_signal_description("FREQUENCY_MAX", "Maximum processor frequency."); // TODO: Remove @ v2.0
+        set_signal_description("CPU_FREQUENCY_MAX", "Maximum processor frequency.");
+    }
+
+    void MSRIOGroup::register_frequency_controls(void) {
+        if (m_hwp_is_enabled) {
+            register_control_alias("FREQUENCY_MIN_CONTROL", "MSR::HWP_REQUEST:MINIMUM_PERFORMANCE"); // TODO: Remove @ v2.0
+            register_control_alias("FREQUENCY_DESIRED_CONTROL", "MSR::HWP_REQUEST:DESIRED_PERFORMANCE"); // TODO: Remove @ v2.0
+            register_control_alias("FREQUENCY_MAX_CONTROL", "MSR::HWP_REQUEST:MAXIMUM_PERFORMANCE"); // TODO: Remove @ v2.0
+            register_control_alias("CPU_FREQUENCY_MIN_CONTROL", "MSR::HWP_REQUEST:MINIMUM_PERFORMANCE");
+            register_control_alias("CPU_FREQUENCY_DESIRED_CONTROL", "MSR::HWP_REQUEST:DESIRED_PERFORMANCE");
+            register_control_alias("CPU_FREQUENCY_MAX_CONTROL", "MSR::HWP_REQUEST:MAXIMUM_PERFORMANCE");
+
+            register_control_alias("PACKAGE_FREQUENCY_MIN_CONTROL", "MSR::HWP_REQUEST_PKG:MINIMUM_PERFORMANCE");
+            register_control_alias("PACKAGE_FREQUENCY_DESIRED_CONTROL", "MSR::HWP_REQUEST_PKG:DESIRED_PERFORMANCE");
+            register_control_alias("PACKAGE_FREQUENCY_MAX_CONTROL", "MSR::HWP_REQUEST_PKG:MAXIMUM_PERFORMANCE");
+        }
+        else {
+            register_control_alias("FREQUENCY_MAX_CONTROL", "MSR::PERF_CTL:FREQ"); // TODO: Remove @ v2.0
+            register_control_alias("CPU_FREQUENCY_MAX_CONTROL", "MSR::PERF_CTL:FREQ");
+        }
```

Comment:
```suggestion
        if (m_hwp_is_enabled) {
            register_control_alias("CPU_FREQUENCY_MAX_CONTROL", "MSR::HWP_REQUEST:MAXIMUM_PERFORMANCE");
        }
        else {
            register_control_alias("CPU_FREQUENCY_MAX_CONTROL", "MSR::PERF_CTL:FREQ");
        }
```
Remove new aliases (such as PACKAGE_FREQUENCY_MAX_CONTROL) unless there is a clear analogue or use case in existing agents